app/assets/stylesheets/landing/languages.scss

```diff
@@ -0,0 +1,19 @@
+.language.footer {
+  height: 26px;
+  padding-bottom: 3px;
+}
+
+.language-button {
+  @media only screen and (max-width: 767px) {
+    position: absolute;
+    right: 2.5em;
+  }
+  @media only screen and (max-width: 230px) {

```

Comment:
please use foundation media queries
`@media #{$small-only} ` or `@media #{$medium-up} `, ecc...see full list here:
http://foundation.zurb.com/docs/media-queries.html in the paragraph 'Style with Sass'